lightly/cli/crop_cli.py

```diff
@@ -0,0 +1,112 @@
+            label_names_file = fix_input_path(label_names_file)
+        with open(label_names_file, 'r') as file:
+            label_names_file_dict = yaml.full_load(file)
+        class_names = label_names_file_dict['names']
+    else:
+        class_names = None
+
+
+    dataset = LightlyDataset(input_dir)
+    filenames_images = dataset.get_filenames()
+    cropped_images_list_list: List[List[Image]] = []
+    for filename_image in filenames_images:
+        filepath_image = dataset.get_filepath_from_filename(filename_image)
+        filepath_image_base, image_extension = os.path.splitext(filepath_image)
+        filepath_label = os.path.join(label_dir, filename_image).replace(image_extension, '.txt')
+        filepath_out_dir = os.path.join(output_dir, filename_image).replace(image_extension, '')
+        Path(filepath_out_dir).mkdir(parents=True, exist_ok=True)
+
+        class_indices, bounding_boxes = read_yolo_label_file(filepath_label, float(cfg['crop_padding']))
+        cropped_images = crop_image_by_bounding_boxes(filepath_image, bounding_boxes)
+        cropped_images_list_list.append(cropped_images)
+        for index, (class_index, cropped_image) in enumerate((zip(class_indices, cropped_images))):
+            if class_names:
+                class_name = class_names[class_index]
+            else:
+                class_name = f"class{class_index}"
+            cropped_image_filename = os.path.join(filepath_out_dir, f'{index}_{class_name}{image_extension}')
+            cropped_image.save(cropped_image_filename)
+
+
+    print(f'Cropped images are stored at: {bcolors.OKBLUE}{output_dir}{bcolors.ENDC}')
+    return cropped_images_list_list
+
+
+
+@hydra.main(config_path="config", config_name="config")
+def crop_cli(cfg):
+    """TODO
+
+    Args:
+        cfg:
+            The default configs are loaded from the config file.
+            To overwrite them please see the section on the config file
+            (.config.config.yaml).
+
+    Command-Line Args:
+        input_dir:
+            Path to the input directory where images are stored.
+        labels_dir:
+            Path to the directory where the labels are stored. There must be one label file for each image.
+            The label file must have the same name as the image file, but the extension .txt.
+            For example, img_123.txt for img_123.jpg. The label file must be in YOLO format.
+        output_dir:
+            Path to the directory where the cropped images are stored. They are stored in one directory per input image.
+        crop_padding: Optional
+            The additonal padding about the bounding box. This makes the crops include the context of the object.
+            The padding is relative and added to the width and height.
+        label_names_file: Optional
+            A yaml file including the names of the classes. If it is given, the filenames of the cropped images include
+            the class names instead of the class id.
```

Comment:
Could you give a minimal example of a yaml file? For example only with two classes, cats and dogs.